Ugly_Number/Nth_Ugly_Number.java

```diff
@@ -0,0 +1,49 @@
+class Ugly{
```

Comment:
Please provide a brief description of what are ugly numbers across all the files at the start of code